python/lsst/pipe/tasks/characterizeImage.py

```diff
@@ -161,11 +161,15 @@ class CharacterizeImageConfig(pipeBase.PipelineTaskConfig,
     doComputeSummaryStats = pexConfig.Field(
         dtype=bool,
         default=True,
-        doc="Run subtask to measure exposure summary statistics"
+        doc="Run subtask to measure exposure summary statistics",
+        deprecated=("This subtask has been moved to CalibrateTask "
+                    "with DM-30701.")
```

Comment:
Do you need a "Will be removed after vNN." declaration here (and below)?

I'm glad you caught this before it may have caused some serious puzzlement!  However, this does mean that anyone running `characterizeImage`, but not `calibrate` will not benefit from getting any summary stats that are appropriately computed in that task (namely the PSF related values).   In fact, I'm now wondering if it actually is more appropriate to actually compute the summary stats in both places (where any values that get updated within `calibrate` would get updated to the new ones?

Reply:
I thought about this a bit, and went back and forth (there is some overhead to compute these twice, but I don't think it's necessarily a burden).  But in the end I landed on this because the `characterizeImage` task will output the `icExp` which really isn't a standard data product, while `calibrate` will output a `calexp` which is.  So if I think of these as "calexp summary statistics" it doesn't need to be in both places.  However, I could be persuaded otherwise.

Reply:
Ah, good point about `icExp` vs. `calexp`.  I do think it's good to leave as is (and if anybody decides they'd really want such summaries on the former, it can be added later -- and optionally, to avoid unnecessary duplication of the computation!)